src/basic/signal-util.h

```diff
@@ -46,6 +46,7 @@ static inline void block_signals_reset(sigset_t *ss) {
                 assert_se(sigprocmask_many(SIG_BLOCK, &_t, __VA_ARGS__) >= 0);     \
                 _t;                                                                \
         })
+#define SIGNO_INVALID (-1)
```

Comment:
for this one maybe define as `(-EINVAL)`

why? because that would nicely match what we return in signal_from_string() if we cannot parse things, and it's otherwise as good or bad as -1